eng/pipelines/templates/steps/test.yml

```diff
@@ -0,0 +1,73 @@
+parameters:
```

Comment:
One thing to think about is that this might be confused between live tests and unit tests. If we can share the logic that would be awesome but if not we might want to think about renaming to avoid confusion.